internal/namespaces/test/v1/test_cli.go

```diff
@@ -350,5 +350,13 @@ func testHumanRun() *core.Command {
 			return api.RunHuman(args)
 
 		},
+		Examples: []*core.Example{
+			{
+				Short: "Create a human and make it run",
+				Raw: `scw test human create
+scw test human run human-id=xxxxx
+`,
```

Comment:
shouldn't the generator remove empty lines at the end ?

Reply:
I created an issue to fix this one on the generator